<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LavAplayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="mid_bg">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>0</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/mid_bg.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="cover">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>22</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/foologo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="mid_cover_mask">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>18</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/mid_ov4.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="top_bg_brush">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>921</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/brushed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="main_bg_color">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>-80</y>
      <width>1143</width>
      <height>860</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(27,27,27);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="right_bg_z0">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>20</y>
      <width>360</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(38,39,41);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_bg_z1">
    <property name="geometry">
     <rect>
      <x>549</x>
      <y>85</y>
      <width>350</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60,65,70);
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_bg_z2">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>90</y>
      <width>344</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(38,39,41);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spectrum_bg_r">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>90</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/bg_spectrum.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="left_bg_z2">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>90</y>
      <width>344</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(38,39,41);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spectrum_bg_l">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>90</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/images/bg_spectrum.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="left_bg_z1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>85</y>
      <width>350</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60,65,70);
border-radius: 4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="left_bg_z0">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>360</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(38,39,41);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="left_spectum_color_mask">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>90</y>
      <width>339</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(27,27,34, 70%);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_spectum_color_mask">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>90</y>
      <width>339</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(27,27,34, 70%);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="button_box_bg">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>161</y>
      <width>220</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40,40,40);
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="noteBtn">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#noteBtn {

	border-image: url(:/pic/images/controls/note0.png);

}

QPushButton#noteBtn:pressed {

	border-image: url(:/pic/images/controls/note1.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#settingsBtn {

	border-image: url(:/pic/images/controls/option0.png);

}

QPushButton#settingsBtn:pressed {

	border-image: url(:/pic/images/controls/option1.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatBtn">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#repeatBtn {

	border-image: url(:/pic/images/controls/po_repeat_pl0.png);

}

QPushButton#repeatBtn:pressed {

	border-image: url(:/pic/images/controls/po_repeat_pl1.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#stopBtn {
	border-image: url(:/pic/images/controls/stop0.png);
}
QPushButton#stopBtn:pressed {
	border-image: url(:/pic/images/controls/stop1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevBtn">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#prevBtn {
	border-image: url(:/pic/images/controls/bt_prev0.png);
}
QPushButton#prevBtn:pressed {
	border-image: url(:/pic/images/controls/bt_prev1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#playBtn {
border-image: url(:/pic/images/controls/bt_play0.png);
}
QPushButton#playBtn:pressed {
	border-image: url(:/pic/images/controls/bt_play1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#nextBtn{
border-image: url(:/pic/images/controls/bt_next0.png);
}
QPushButton#nextBtn:pressed{
border-image: url(:/pic/images/controls/bt_next1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="likeBtn">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#likeBtn {

	border-image: url(:/pic/images/controls/bt_mood0on.png);

}

QPushButton#likeBtn:pressed {

	border-image: url(:/pic/images/controls/bt_mood1on.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="skipBtn">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton#skipBtn {

	border-image: url(:/pic/images/controls/bt_skip0on.png);

}

QPushButton#skipBtn:pressed {

	border-image: url(:/pic/images/controls/bt_skip1on.png);

}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="play_list_bg">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>215</y>
      <width>901</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(95,100,105);
border-radius: 6px;
background-color: qlineargradient(spread:pad, x1:0.437811, y1:0, x2:0.483299, y2:1, stop:0.512438 rgba(95, 100, 105, 255), stop:1 rgba(50, 52, 54, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="play_list_bg_z1">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>220</y>
      <width>891</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 20, 20);
border-radius: 6px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="spectrum_bg_down">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>530</y>
      <width>741</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 39, 39);
border-radius: 6px;
border: 2px solid #000000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timePassed">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>25</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:20, stop:0.512438 rgba(72, 75, 78, 255), stop:1 rgba(136, 140, 145, 255));
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="timePassed_back">
    <property name="geometry">
     <rect>
      <x>553</x>
      <y>25</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgba(50,50,50,100%);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="timePassedTotal">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>25</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:20, stop:0.512438 rgba(72, 75, 78, 255), stop:1 rgba(136, 140, 145, 255));
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="timePassedTotal_back">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>25</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: rgba(50,50,50,100%);
}</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="format">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>68</y>
      <width>111</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(119, 147, 54);
background-color: qlineargradient(spread:pad, x1:0.437811, y1:0, x2:0.483299, y2:1, stop:0.512438 rgba(51, 50, 51, 255), stop:1 rgba(69, 70, 71, 255));
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bit_rate">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>68</y>
      <width>111</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(119, 147, 54);
background-color: qlineargradient(spread:pad, x1:0.437811, y1:0, x2:0.483299, y2:1, stop:0.512438 rgba(51, 50, 51, 255), stop:1 rgba(69, 70, 71, 255));
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string>BitRate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sample_rate">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>68</y>
      <width>111</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(119, 147, 54);
background-color: qlineargradient(spread:pad, x1:0.437811, y1:0, x2:0.483299, y2:1, stop:0.512438 rgba(51, 50, 51, 255), stop:1 rgba(69, 70, 71, 255));
border-radius: 2px;</string>
    </property>
    <property name="text">
     <string>SampleRate</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="percent">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>26</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:20, stop:0.512438 rgba(51,54,55, 255), stop:1 rgba(72, 75, 78, 255));
}</string>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="process_bar">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>47</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
background: transparent;
border: 1px dotted  rgb(73,74,82);
height: 6px;
}

QSlider::sub-page:horizontal {
background:transparent;
height: 3px;
}

QSlider::add-page:horizontal {
background: transparent;
height: 3px;
}

QSlider::handle:horizontal {
background: rgb(80,80,80);
width: 6px;
}

QSlider::handle:horizontal:hover {
background: rgb(153,153,153);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="artist_label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>26</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
    </property>
    <property name="text">
     <string>Artist</string>
    </property>
   </widget>
   <widget class="QLabel" name="album_label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>46</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>66</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="artist">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>26</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:8, stop:0.512438 rgba(72, 75, 78, 255), stop:1 rgba(136, 140, 145, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="album">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>46</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:8, stop:0.512438 rgba(72, 75, 78, 255), stop:1 rgba(136, 140, 145, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>66</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
/*color: rgb(72,75,76);*/
	color: qlineargradient(x1:0.437811, y1:0, x2:0.483299, y2:8, stop:0.512438 rgba(72, 75, 78, 255), stop:1 rgba(136, 140, 145, 255));
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseBtn">
    <property name="geometry">
     <rect>
      <x>138</x>
      <y>163</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pauseBtn {
border-image: url(:/pic/images/controls/bt_pause0.png);
}
QPushButton#pauseBtn:pressed {
	border-image: url(:/pic/images/controls/bt_pause1.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>main_bg_color</zorder>
   <zorder>top_bg_brush</zorder>
   <zorder>mid_bg</zorder>
   <zorder>cover</zorder>
   <zorder>mid_cover_mask</zorder>
   <zorder>right_bg_z0</zorder>
   <zorder>right_bg_z1</zorder>
   <zorder>right_bg_z2</zorder>
   <zorder>spectrum_bg_r</zorder>
   <zorder>left_bg_z0</zorder>
   <zorder>left_bg_z1</zorder>
   <zorder>left_bg_z2</zorder>
   <zorder>spectrum_bg_l</zorder>
   <zorder>left_spectum_color_mask</zorder>
   <zorder>right_spectum_color_mask</zorder>
   <zorder>button_box_bg</zorder>
   <zorder>noteBtn</zorder>
   <zorder>settingsBtn</zorder>
   <zorder>repeatBtn</zorder>
   <zorder>stopBtn</zorder>
   <zorder>prevBtn</zorder>
   <zorder>playBtn</zorder>
   <zorder>nextBtn</zorder>
   <zorder>likeBtn</zorder>
   <zorder>skipBtn</zorder>
   <zorder>play_list_bg</zorder>
   <zorder>play_list_bg_z1</zorder>
   <zorder>spectrum_bg_down</zorder>
   <zorder>timePassed_back</zorder>
   <zorder>timePassed</zorder>
   <zorder>timePassedTotal_back</zorder>
   <zorder>timePassedTotal</zorder>
   <zorder>format</zorder>
   <zorder>bit_rate</zorder>
   <zorder>sample_rate</zorder>
   <zorder>percent</zorder>
   <zorder>process_bar</zorder>
   <zorder>artist_label</zorder>
   <zorder>album_label</zorder>
   <zorder>title_label</zorder>
   <zorder>artist</zorder>
   <zorder>album</zorder>
   <zorder>title</zorder>
   <zorder>pauseBtn</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
